<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNuevaCasa</class>
 <widget class="QDialog" name="dlgNuevaCasa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNombre"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Lugar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="myQLineEdit" name="txtLugar"/>
      </item>
      <item>
       <widget class="QPushButton" name="btQuitarLugar">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Lugar originario</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtLugarOriginario">
        <property name="text">
         <string>Lugar originario</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Provincia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="myQLineEdit" name="txtProvincia"/>
      </item>
      <item>
       <widget class="QPushButton" name="btQuitarProvincia">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="ckBuscado">
        <property name="text">
         <string>Buscado</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckMasculino">
        <property name="text">
         <string>Masculino</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckWiki">
        <property name="text">
         <string>Wikipedia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckStudium">
        <property name="text">
         <string>Studium generale</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="txtNotas"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btCancelar">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btOK">
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/myqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
